### PR TITLE
PMM-8460: service software versions

### PR DESCRIPTION
**Pull request by oter**
https://jira.percona.com/browse/PMM-8460

agent PR: https://github.com/percona/pmm-agent/pull/283
managed PR: https://github.com/percona/pmm-managed/pull/807

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1941

### CODE REVIEW

**Review by BupycHuk — DISMISSED** [review]

File: api/agentpb/agent.proto [hunk 1]
```diff
@@ -534,6 +534,38 @@ message JobProgress {
...
+      Qpress qpress = 4;
+    }
+  }
+  repeated Software softwares = 1;
+}
+
+// GetVersionResponse represents a response containing versions of software in the same order as in the request.
+message GetVersionResponse {
+  repeated string versions = 1;
+  repeated string errors = 2;
```
Comment (Dasio) [inline api/agentpb/agent.proto:566]: I assume if `versions[0] == ""` -> `errors[0] != ""` Isn't better to couple this to one message?

Comment (oter) [reply api/agentpb/agent.proto]: yeah, coupled to the one message

**Review by Dasio — DISMISSED** [review]

**Review by artemgavrilov — APPROVED** [review]

File: api/agentpb/agent.go [hunk 2]
```diff
@@ -41,7 +41,8 @@ type ServerRequestPayload interface {
 	sealed()
 }
 
-// AgentMessage request payloads
+// A list of AgentMessage request payloads
```
[the remark this answers is not in the document]
Comment (oter) [reply api/agentpb/agent.go]: dots added

**Review by artemgavrilov — DISMISSED** [review]
Lol, I thought I approved this PR)

**Comment by Dasio** [comment]
> Lol, I thought I approved this PR)

it was auto dismissed by commit

**Review by Dasio — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by Dasio — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**